windows/tests/9.3.x/README.md

```diff
@@ -1,3 +1,57 @@
 # Tests
 
-...
\ No newline at end of file
+# Running XC
```

Comment:
add license info, both license keys environments variables are required to run XC